translations/fr.json

```diff
@@ -171,15 +171,24 @@
   "domain.record.status.ongoing": "Mise à jour continue",
   "domain.record.status.removed": "Supprimé",
   "domain.record.status.under_development": "En cours de création",
+  "domain.record.updateFrequency.annually": "Annuelle",
   "domain.record.updateFrequency.asNeeded": "Lorsque nécessaire",
+  "domain.record.updateFrequency.biannually": "Bi-annuelle",
+  "domain.record.updateFrequency.biennially": "Tous les 2 ans",
```

Comment:
To me, it would be:
biannually => "Semestrielle"
biennially => "Bisannuelle"

Reply:
Ask PO :smile: